templates/country-list.html

```diff
@@ -6,8 +6,10 @@
 
 {{ define "main" }}
 <main class="container">
-    <h1>Countries</h1>
-
+    <div>
+        <a href="/map/" class="btn right" style="float:right">View on a map</a>
```

Comment:
The inline style can be removed as the code is also in the stylesheet.

The URL should either be `map/` or `/countries/map/`.